trickuri.go

```diff
@@ -322,7 +323,7 @@ func httpHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// An index explaining how to use the different testcases.
-	http.ServeFile(w, r, "index.html")
+	http.ServeFile(w, r, *testcaseDirectory+"index.html")
```

Comment:
I think you also need to do something similar in the line where http.FileServer is initialized for /web-feature-tests (line 320).

Reply:
done